src/rebar3_typer_prv.erl

```diff
@@ -3,6 +3,8 @@
 
 -export([init/1, do/1, format_error/1]).
 
+-define(PRV_ERROR(Reason), {error, {?MODULE, Reason}}).
```

Comment:
If you're going to go this way… include the corresponding hrl. Otherwise, there's almost no reason to have a macro instead of using the tuples directly in the code.